src/materialized/bin/materialized.rs

```diff
@@ -120,6 +120,11 @@ fn run() -> Result<(), failure::Error> {
     );
     opts.optopt("", "symbiosis", "(internal use only)", "URL");
     opts.optflag("", "no-prometheus", "do not gather prometheus metrics");
+    opts.optflag(
+        "",
+        "disable-sink-suffix",
+        "(internal use only) do not add suffix to sink storange name",
+    );
```

Comment:
I would really prefer not to add all this complexity, even temporarily. I think we should either tackle the the `SHOW SINKS` bit in this PR, or just comment out the one `kafka-verify` call in testdrive for now.

Reply:
so the other thing is that even if we added the show sinks bit, my understanding is that testdrive is currently unable to set a variable to be the result of a sql query / variables can only be assigned static strings? is that accurate?

Reply:
Regardless, its fairly easy to rip this out for now and address in a follow on if you are ok with ripping out the kafka verify for now